src/bp/ui-studio/src/web/components/ContentForm/index.tsx

```diff
@@ -57,7 +57,7 @@ const CustomDescriptionField = ({ description, id, formContext }) => {
     return (
       <div id={id} style={{ lineHeight: 'normal' }}>
         <div>
-          <span style={{ color: Colors.ORANGE3 }}>
+          <span style={{ color: 'var(--lighthouse)' }}>
```

Comment:
can you use a class for this 🙏 
Also change the icon warning to lighthouse por favor